### ISSUE TITLE
Unable to test for SublimeText 3

### ISSUE THREAD

**Issue by kaste** [SublimeText/UnitTesting#190]
It looks like that we can't test against Sublime Text 3. 

For example, just re-running the test from `master` gives https://github.com/kaste/UnitTesting/pull/1/checks?check_run_id=2684397425

**Comment by randy3k** [comment]
Um, it used to work a few days ago. Let me take a look later.

**Comment by randy3k** [comment]
I haven’t checked, but I have a feeling that it is due to a pop up prompting ST3 users to upgrade to ST4.

**Comment by randy3k** [comment]
Confirmed.
![Screen Shot 2021-05-27 at 8 04 59 AM](https://user-images.githubusercontent.com/1690993/119850371-542b7880-bec2-11eb-8c1e-7c939a967ecd.png)

**Comment by randy3k** [comment]
The setting
```
"update_check": false
```
does work but it requires us to specify a license that is not very feasible in CI.

**Comment by kaste** [comment]
Hm, appveyor still passes though https://github.com/timbrel/GitSavvy/pull/1476

**Comment by kaste** [comment]
Can't we just send `ESC` to the dialog window?

**Comment by kaste** [comment]
Or we block the Sublime ip addr.  That should be doable on linux

**Comment by randy3k** [comment]
> Hm, appveyor still passes though [timbrel/GitSavvy#1476](https://github.com/timbrel/GitSavvy/pull/1476)

AFAIK, the popup on macOS and Linux is blocking the UI, perhaps the popup is not blocking on windows.

**Comment by kaste** [comment]
Ah, the popup is not blocking on Windows.  You can just `alt+tab` to the editor window.

**Comment by randy3k** [comment]
> Can't we just send `ESC` to the dialog window?

> Or we block the Sublime ip addr. That should be doable on linux

Blocking Sublime IP sounds a better approach. Sending ESC is difficult to be delivered consistently and cross platformly.

**Comment by randy3k** [comment]
The following does work for [macos](https://github.com/SublimeText/UnitTesting/runs/2689427595?check_suite_focus=true)
```
sudo route -n add -host 45.55.41.223 127.0.0.1
```
however its linux variate doesn't work because `route` seemingly doesn't work in docker containers without additional privilege.

**Comment by randy3k** [comment]
```
New-NetFirewallRule -DisplayName "Block sublimetext.com IP address" -Direction Outbound –LocalPort Any -Protocol TCP -Action Block -RemoteAddress 45.55.41.223
```
works on windows.

**Comment by randy3k** [comment]
Ironically, I still cannot find a way to block ip on docker without "--privileged".

**Comment by randy3k** [comment]
I found a solution for Linux, but it requires passing `--cap-add=NET_ADMIN` to docker.
https://github.com/SublimeText/UnitTesting/blob/blockip/.github/workflows/build.yml#L16-L17

**Comment by randy3k** [comment]
Fixed in master.

TLDR:  add `--cap-add=NET_ADMIN` for linux and nothing for macos and windows.

**Comment by kaste** [comment]
Super powers, randy, really.  🥇 💪

**Comment by kaste** [comment]
Maybe make a new release.
